cms/tests/test_page_user_admin.py

```diff
@@ -393,7 +396,13 @@ def test_user_cant_change_self(self):
 
         with self.login_user_context(staff_user):
             response = self.client.post(endpoint, data)
-            self.assertEqual(response.status_code, 404)
+            if DJANGO_1_10:
+                self.assertEqual(response.status_code, 404)
+            else:
+                self.assertRedirects(response, redirect_to)
+                msgs = CookieStorage(response)._decode(response.cookies['messages'].value)
```

Comment:
The post might be redirected for different reasons, and it would be better to track it